src/x11-calc.in

```diff
@@ -159,14 +159,16 @@ _config (){
 
    case $? in
       0)
+         _tmpfile="$(mktemp)"
          if [ "$DROPDOWN" ] # The seperator used by different forms is different!
          then
-            sed -i "s|^MODEL=.*|MODEL=\"${_selection%|*}\"|" "$_f_conf"
-            sed -i "s|^OPTIONS=.*|OPTIONS=\"${_selection#*|}\"|" "$_f_conf"
+            sed "s|^MODEL=.*|MODEL=\"${_selection%|*}\"|" "$_f_conf" > $_tmpfile
```

Comment:
I'm not sure this is necessary as `zenity` is very unlikely to be available on systems that don't support `sed -i` and I think it would be better to avoid using a temporary file..?

Reply:
may clash on darwin...

Reply:
Well spotted  NetBSD allows `sed -i'' 's/there/world/g' test` but OSX appears to need the space.